docs/usage/configuration-options.md

```diff
@@ -1663,10 +1663,37 @@ See also `matchPackagePrefixes`.
 
 The above will match all package names starting with `eslint` but exclude ones starting with `eslint-foo`.
 
+### matchCategories
+
+Use `matchCategories` to restrict rules to a particular language or group.
+The categories can be found in the [manager documentation](./modules/manager.md).
+
+<!-- prettier-ignore -->
+!!! warning
+    Rules with `matchCategories` are only applied _after_ extraction of a repository.
+    You can not use `matchCategories` to stop processing a repository.
+    You can still use `matchCategories` to stop or change updating the dependencies though.
```

Comment:
```suggestion
!!! note
    Rules with `matchCategories` are only applied _after_ extraction of dependencies.
    You can't use `matchCategories` to prevent particular categories from being extracted, but you can use it after extraction to prevent them from being looked up or updated.
```